### DISCUSSION TITLE
Sprayer boom height control

### DISCUSSION THREAD

**Discussion by polosubtil** [Courseplay/Courseplay_FS22#1165] [Ideas]
Hi guys,

Just an idea just in case you come across an easy way to do it...

With FS22, sprayer booms are now following the ground, which is awesome. Using the ToolPositionSaver, I can easly set up 2 heights which are working really while spraying myself.
But when I engage either the game helper or CP, the booms goes back to the game preset height immediately. I guess this is game related and has nothing to do with CP.

A great feature to make it look nicer would be for CP to use the preset height at work, using the ToolPositionSaver (or the height set manually by the player)
An even amazing feature would be to lift the boom to a second ToolPositionSaver (higher) in turns... ;)

Not urgent or anything of course, there is a long list of stuff to do before that, just to have it somewhere... just in case :)

**Comment by Tensuko** [comment]
We let Giants control that.
Not something we rly have to control by CP.
